pkg/platform/api/buildplanner/response/commit.go

```diff
@@ -133,9 +133,9 @@ type TargetNotFound struct {
 	PossibleTargets []string `json:"possibleTargets"`
 }
 
-type ValidationError struct {
-	SubErrors []*BuildExprError `json:"subErrors"`
-}
+// ValidationError represents a validation error that occurred during planning.
+// Contains message and subErrors which is handled separately
+type ValidationError struct{}
```

Comment:
We now have multiple errors with `SubErrors` so I moved that to the common `Error` struct